.github/workflows/stage.yml

```diff
@@ -1,35 +0,0 @@
-name: Check links and stage
-
-on:
-  pull_request:
-    branches:
-      - main
-
-# Declare default permissions as read only.
-permissions: read-all
-
-jobs:
-  stage:
-    permissions:
-      contents: read
-      checks: write
-      pull-requests: write
-    runs-on: ubuntu-latest
-    if: ${{ github.ref != 'refs/heads/main' }}
-    steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
-        with:
-          submodules: recursive
-      - name: Build site and check links
-        run: make build BUILD_CONFIGS=_config.yml,_config_stage.yml
-      - name: Stage site on Firebase
```

Comment:
So with the current cloud build template I moved everything from this workflow, including checking the links (see [here](https://github.com/flutter/website/pull/9702/files#diff-0b4a5fc26445836e6672c426692297004cfaba21997343b63e6f58b946bd2a5cR10)), since that step is needed to deploy the website, right? Or am I misunderstanding how this workflow works?

Reply:
Good point! I also didn't think about the fact that keeping a separate GitHub Action for check links also means the site gets built twice. Let's keep it in Cloud Build!